torch/nn/parallel/__init__.py

```diff
@@ -4,6 +4,8 @@
 from .scatter_gather import scatter, gather
 from .distributed import DistributedDataParallel
 from .distributed_cpu import DistributedDataParallelCPU
+from .distributed_c10d import DistributedDataParallelC10d
 
 __all__ = ['replicate', 'scatter', 'parallel_apply', 'gather', 'data_parallel',
-           'DataParallel', 'DistributedDataParallel', 'DistributedDataParallelCPU']
+           'DataParallel', 'DistributedDataParallel', 'DistributedDataParallelCPU',
+           'DistributedDataParallelC10d']
```

Comment:
No, this struct should not appear here. Ideally it would be private (prefixed with an underscore). We never want to have `DistributedDataParallelC10d` as part of our public API, we'll need to replace the regular `DistributedDataParallel` once c10d becomes our main distributed backend.